provisioner/aws-manager.js

```diff
@@ -68,9 +68,7 @@ module.exports = AwsManager;
  * Update the state from the AWS API and return a promise
  * with no resolution value when completed.
  */
-AwsManager.prototype.update = function () {
-  var that = this;
-
+AwsManager.prototype.update = async function () {
```

Comment:
I'm curious, but why not go all the way and convert to `class` style constructs?